api/opencl_platform_layer.asciidoc

```diff
@@ -1483,6 +1483,13 @@ include::{generated}/api/version-notes/CL_DEVICE_PREFERRED_WORK_GROUP_SIZE_MULTI
         (Refer also to {clGetKernelWorkGroupInfo} where {CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE}
         can return a different value to {CL_DEVICE_PREFERRED_WORK_GROUP_SIZE_MULTIPLE} which may be more optimal.)
 
+| {CL_DEVICE_LATEST_CTS_VERSION_PASSED_anchor}
+
+include::{generated}/api/version-notes/CL_DEVICE_LATEST_CTS_VERSION_PASSED.asciidoc[]
+  | char[]
+      | Returns the latest version of the conformance test suite that this device
```

Comment:
Do we need to state "latest" here (and in the enums)?

I think it should only matter about reporting the specific version that the implementation was tested against.

Reply:
I think the intention was to the report "the latest version that was tested", which may not be "the latest version that exists".

I tried a few rewrites in my head, but they didn't seem better than the current wording.

Reply:
>I tried a few rewrites in my head, but they didn't seem better than the current wording.

Thats a good exercise and I also can't come up with a meaningful change either.

Reply:
The analogous Vulkan physical device property is [conformanceVersion](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VkConformanceVersion), so if we wanted to be similar in OpenCL we could consider `CL_DEVICE_CONFORMANCE_VERSION`.  This also has the slight advantage of not requiring the abbreviation `CTS`, which may not be universally known.

I don't have a very strong opinion about this and I'm fine keeping `CL_DEVICE_LATEST_CTS_VERSION_PASSED`.

Reply:
I find `CL_DEVICE_CONFORMANCE_VERSION` easier to read, but I also don't have a strong opinion.